nexus/api/graphql/mutation_create_run_files.graphql

```diff
@@ -0,0 +1,10 @@
+mutation CreateRunFiles($entity: String!, $project: String!, $run: String!, $files: [String!]!) {
```

Comment:
this is the more modern version of:
query_upload.graphql
does it make sense to update and remove this graphql until we have versioning?
I think i was a chicken and just did the lowest support rather than using the more modern schema.

Reply:
looks like you already removed references to the old Upload call. can you remove the .graphql file also if it isnt being used anymore